src/arguments/boolean.ts

```diff
@@ -0,0 +1,18 @@
+import { Argument, ArgumentStore, KlasaMessage, Possible } from 'klasa';
+const truths = ['1', 'true', '+', 't', 'yes', 'y'];
+const falses = ['0', 'false', '-', 'f', 'no', 'n'];
```

Comment:
maybe those 2 should be exported or added as static properties to the Argument

Reply:
Not changing existing behavior at this time. You can still transfer and edit this piece.